src/components/Products/ProductTile/ProductTile.js

```diff
@@ -1,21 +1,21 @@
 import React from 'react'
-import './ProductTile.scss'
+import './productTile.scss'
 
-function ProductTile(props) {
+function productTile(props) {
     let ProductRender = (products) => {
         return products.map(product => <div className="product"><a href="#">
             <img src={product.ImgUrl} alt="" width="100" height="100" /></a>
             <div className="product-description">
-                       Name : {product.Name}
+                Name : {product.Name}
                 <br />Brand : {product.Brand}
                 <br />Price : {product.Price}
             </div></div>);
     }
-    
+
     return (<div className={props.customClass}>
         {ProductRender(props.products)}
     </div>)
 
 }
 
-export default ProductTile
\ No newline at end of file
+export default productTile
```

Comment:
Class name should be ProductTile but file name should be productTile.js

Reply:
changed to ProductTile